.github/workflows/spellcheck.yml

```diff
@@ -19,3 +19,7 @@ jobs:
         # typo_ci_license_key: ${{ secrets.TYPO_CI_LICENSE_KEY }}
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+# Any files/folders we should ignore?
```

Comment:
maybe ignoring all `*.py` files would be neat as well (you can maybe take the `spellcheck.yml` from the Rasa Open Source repository as inspiration). This goes beyond the scope of this PR so feel free to just create an issue for this or do it in a separate PR

Reply:
Maybe you mean the [typo-ci.yml](https://github.com/RasaHQ/rasa/blob/main/.typo-ci.yml) that Melinda mentioned previously and contains the `*.py` that you are saying? Because in Rasa-OSS there is : 
- [typo-ci.yml](https://github.com/RasaHQ/rasa/blob/main/.typo-ci.yml) which has these type of file checks and
- [spellcheck.yml](https://github.com/RasaHQ/rasa/blob/main/.github/workflows/spellcheck.yml) which does not contain file related checks. 

Now that I see this again I think I did something wrong because it seems there are different files for different things : 
- `spellcheck.yml` as it is described [here](https://github.com/marketplace/actions/github-spellcheck) is "_...for spell checking in GitHub Actions._"
- `typo-ci.yml` as it is described [here](https://typoci.com/documentation) is for "_...checks Pull Requests for spelling mistakes every time you push your code up to GitHub_"

and I copied a file check / rule (`"rasa_sdk/**/*"`) from `typo-ci.yml` (rasa-oss) to `spellcheck.yml` (rasa-sdk) because in rasa-sdk there is no typo-ci file (at least I could not find one). But maybe that is not correct and I should create a typo-ci.yml in the root folder of rasa-sdk and just copy all the checks/rules from rasa-oss as Melinda suggested.

Reply:
good catch!

Reply:
>  My only concern is why this file was not already there? Was it removed for a reason and I should not add it?

I guess nobody ran into this 🤷🏻